tests/foreman/ui/test_repository.py

```diff
@@ -714,6 +714,35 @@ def test_positive_sync_ansible_collection_gallaxy_repo(session, module_prod):
         assert result['result'] == 'success'
 
 
+@pytest.mark.tier2
+def test_positive_no_errors_on_repo_scan(target_sat):
+    """Scan repos for RHEL Server Extras, then check the production log
+    for a specific error
+
+    :id: 443bf4af-7f9a-48b8-8f98-fdb170e8ae88
+
+    :expectedresults: The specific error isn't contained in the prod log
+
+    :customerscenario: True
+
+    :BZ: 1994212
+
+    :CaseLevel: Integration
+    """
+    org = target_sat.api.Organization().create()
+    manifests.upload_manifest_locked(org.id)
+    sat_rpm_extras = target_sat.cli_factory.RHELServerExtras(cdn=True)
+    with target_sat.ui_session() as session:
+        session.organization.select(org.name)
+        session.redhatrepository.read(sat_rpm_extras.data['repository-set'])
+        result = target_sat.execute(
+            'cat /var/log/foreman/production.log | '
+            'grep "Failed at scanning for repository: undefined method '
+            '`resolve_substitutions\' for nil:NilClass"'
+        )
+        assert result.status == 1
```

Comment:
Non-blocking, but I would add some descriptive message to the user in case the assertion fails.